runtime/frequency/build.rs

```diff
@@ -1,12 +1,40 @@
+// Duplicated here to reduce the build dependencies
+#[allow(unused)]
+const FREQUENCY_TESTNET_TOKEN: &str = "XRQCY";
+#[allow(unused)]
+const FREQUENCY_LOCAL_TOKEN: &str = "UNIT";
+#[allow(unused)]
+const FREQUENCY_TOKEN: &str = "FRQCY";
+#[allow(unused)]
+const TOKEN_DECIMALS: u8 = 8;
+
 #[cfg(all(feature = "std", not(feature = "metadata-hash")))]
 fn main() {
 	substrate_wasm_builder::WasmBuilder::build_using_defaults()
 }
 
-#[cfg(all(feature = "std", feature = "metadata-hash"))]
+#[cfg(all(feature = "std", feature = "metadata-hash", feature = "frequency"))]
+fn main() {
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash(FREQUENCY_TOKEN, TOKEN_DECIMALS)
+		.build()
+}
+
+#[cfg(all(feature = "std", feature = "metadata-hash", feature = "frequency-testnet"))]
+fn main() {
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash(FREQUENCY_TOKEN, TOKEN_DECIMALS)
```

Comment:
```suggestion
		.enable_metadata_hash(FREQUENCY_TESTNET_TOKEN, TOKEN_DECIMALS)
```

Reply:
Thank you. The one thing I am trying to fix and my copy paste for the final setup broke it!